scripts/package-info/devnet.json

```diff
@@ -10,13 +10,13 @@
         }
     },
     "iotaNamesObjectId": "0x07c59b37bd7d036bf78fa30561a2ab9f7a970837487656ec29466e817f879342",
-    "packageId": "0xb9d617f24c84826bf660a2f4031951678cc80c264aebc4413459fb2a95ada9ba",
+    "packageId": "0xd8d28a7446a93b1cc0ae248d19b649b9f9b5e0b05332a31aa3ab514e4b7aff32",
```

Comment:
Actually not sure about this change here, since the types still have the old version
We might need to change it so we can have multiple ids

Reply:
Yeah I wasn't sure either 😬